tests/integration/conftest.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+
+from cluster_fixture import ClusterFixture
+
+
```

Comment:
Is this a test for ClusterFixture (if so, what is the test pass criteria?), or a helper method for the tests (if so, the name conftest seems wrong)?

Docstrings.

Reply:
not sure I follow it brings ClusterFixture into scope. It looks odd because we don't have an __init__.py but that is per the pytest documentation and necessary to avoid issues importing the c extension.

Reply:
I don't understand what the purpose of the method is: is it a utility method used by the tests, or is it a test itself?
The filename suggests that it is a config test, but I don't see any verification criteria, which leads me to believe it is a utility function used by the tests, but the filename confuses me.